pkg/webreg/webreg.go

```diff
@@ -729,6 +729,78 @@ to configure that test to run on a schedule, instead of as a pre-submit:
 Note that the build farms used to execute jobs run on UTC time, so time-of-day based
 <code>cron</code> schedules must be set with that in mind.
 </p>
+
+<h3 id="image-references"><a href="#image-references">Referencing Images</a></h3>
+<p>
+As <code>ci-operator</code> is OpenShift-native, all images used in a test workflow
+are stored as <code>ImageStreamTags</code>. Inside of the <code>Namespace</code> which
```

Comment:
This reads like you replaced a comma with a dot because the sentence got to long. What about reversing the second part of the sentence to:
`The following imageStreams will exist in the Namespace which contains a test workflow:`

Reply:
I had the exact same thought while reading this.

Reply:
My brain is weird. I wrote it like this from the get-go